package.json

```diff
@@ -16,25 +27,41 @@
     "scheduler",
     "processes"
   ],
-  "author": "Daniel Kostro <kostro.d@gmail.com>",
+  "author": "Daniel Kostro",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/cheminfo/process-scheduler/issues"
   },
   "homepage": "https://github.com/cheminfo/process-scheduler#readme",
-  "dependencies": {
-    "debug": "^3.1.0",
-    "node-schedule": "^1.3.0"
-  },
   "jest": {
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json",
+      "node"
+    ]
   },
   "devDependencies": {
-    "eslint": "^5.4.0",
-    "eslint-config-cheminfo": "^1.17.3",
-    "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-jest": "^21.15.1",
+    "@types/debug": "0.0.30",
```

Comment:
I would put them in dev as well since I suppose the compiled source code does depend on them. But I noticed that you put them in production dependencies in another project. Is there a reason for that?

Reply:
I try to put in prod only the ones that can be accessed by the end user (for example a type returned by our Api)